<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>XIVN1987</author>
 <class>RTTView</class>
 <widget class="QWidget" name="RTTView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>560</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SEGGER-RTT Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Image/serial.ico</normaloff>Image/serial.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="vLayout">
   <item>
    <widget class="QTextEdit" name="txtMain">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gLayout1">
     <item row="0" column="3">
      <widget class="QPushButton" name="btnOpen">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>打开连接</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblDLL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>DLL ：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbDLL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbAddr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnDLL">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblAddr">
       <property name="text">
        <string>Addr：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QCheckBox" name="chkWave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>波形显示</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnClear">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>清除显示</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnAddr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QCheckBox" name="chkSave">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>保存接收</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gLayout2">
     <item row="0" column="1" rowspan="5">
      <widget class="QPushButton" name="btnSend">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="cmbEnter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>发送回车编码</string>
       </property>
       <property name="toolTipDuration">
        <number>1000</number>
       </property>
       <item>
        <property name="text">
         <string>\r\n</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>\n</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QComboBox" name="cmbSpeed">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>1 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>40 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50 MHz</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>80 MHz</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cmbOCode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>发送内容编码</string>
       </property>
       <property name="toolTipDuration">
        <number>1000</number>
       </property>
       <item>
        <property name="text">
         <string>ASCII</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HEX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GBK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTF-8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0" rowspan="5">
      <widget class="QTextEdit" name="txtSend">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="cmbICode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>接收内容编码</string>
       </property>
       <property name="toolTipDuration">
        <number>1000</number>
       </property>
       <item>
        <property name="text">
         <string>ASCII</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HEX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GBK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTF-8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="cmbMode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>ARM SWD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ARM JTAG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RV cJTAG</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RV JTAG</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblVar">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <column/>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
